coolwsd.xml.in

```diff
@@ -195,10 +194,6 @@
       <enable_metrics_unauthenticated desc="When enabled, the /cool/getMetrics endpoint will not require authentication." type="bool" default="false">false</enable_metrics_unauthenticated>
     </security>
 
-    <certificates>
```

Comment:
This is unrelated, and important to keep. User can set NSS database to validate signatures. Without this, signatures cannot be validated.

Reply:
Thanks for the review :) Done.